src/core/evaluator.js

```diff
@@ -777,7 +777,15 @@ class PartialEvaluator {
       });
   }
 
-  handleSetFont(resources, fontArgs, fontRef, operatorList, task, state) {
+  handleSetFont(
+    resources,
+    fontArgs,
+    fontRef,
+    operatorList,
+    task,
+    state,
+    fallbackFontDict
```

Comment:
Please indicate that the parameter is optional here, i.e.
```suggestion
    fallbackFontDict = null
```